common/lib/xmodule/xmodule/js/src/video/09_play_skip_control.js

```diff
@@ -28,12 +28,7 @@ define('video/09_play_skip_control.js', [], function() {
             '<button class="control video_control play play-skip-control" title="',
                 gettext('Play'),
             '">',
-                '<span class="icon-fallback-img">',
-                    '<span class="icon fa fa-play" aria-hidden="true"></span>',
-                    '<span class="text control-text">',
```

Comment:
Since you are removing this control-text span, you also need to remove its associated JavaScript.